docs/install/index.rst

```diff
@@ -37,17 +41,54 @@ To check your Gammapy installation, you can use this command:
 
     gammapy info
 
-Congratulations! You are all set to start using Gammapy!
 
-If you're new to Python, ipython and Jupyter, read the :ref:`getting-started`
-guide. To learn how to use Gammapy, go to :ref:`tutorials`.
+Using other package managers
+----------------------------
 
 The following pages contain detailed information about Gammapy dependencies in
```

Comment:
Maybe mention something like "If you don't want to use Anaconda, you can...".
This makes it clearer to python novices that once the installation is successful using conda, there is no requirement to go through this sub-section or bother about dependencies explicitly.

Reply:
Good point. Added